travis/build-and-test.sh

```diff
@@ -34,3 +34,13 @@ make -j$(nproc)
 
 # run all unit tests
 ctest -V
+
+# install libappimage
+DESTDIR=$TEMP_BASE/libappimage make install
```

Comment:
Can we move this into a separate script, please? Then, it can be developed and tested separately.

Reply:
This section requires the whole project to be built and installed. If we move it to another script we will have to rebuilt the project or pass the build path as an argument which will be pointless.

Reply:
Why does the build path have to be passed to the script? It's executed there... so the CWD will be that path...

Reply:
Moving just those lines to a separated script that can only been executed from here makes a very fragile script. Someone may try to run it the wrong way and will came back complaining that it wont work.

Reply:
I don't insist on it. I just figured it'd be nice to be able to debug this part without having to run a full build. But you're right, the script itself would be quite useless, as some setup is required for it anyway.